src/acceptance/assets/file/policy/policy-with-credential-type.json

```diff
@@ -0,0 +1,81 @@
+{
+  "credential_type": "x509",
```

Comment:
FYI: credential_type param is snake case (instead of kebab-case) due to current naming in the policy json

Reply:
renamed to `credential-type`